src/blocks/author-grid/components/AdvancedStyling.js

```
@@ -0,0 +1,164 @@
+				checked={ gradientBackground }
+				onChange={ ( value ) => onChange( { gradientBackground: value } ) }
+				help={ __(
+					'Apply a gradient background to grid items',
+					'author-profile-blocks',
+				) }
+			/>
+
+			{ gradientBackground && (
+				<>
+					<div style={ { marginBottom: '16px' } }>
+						<label style={ { display: 'block', marginBottom: '8px', fontWeight: '600' } }>
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)